app/presenters/help_page_presenter.rb

```diff
@@ -0,0 +1,2 @@
+class HelpPagePresenter < GovspeakPresenter
```

Comment:
>  perhaps we should be putting the tickets in now?
Never a reason not to put a ticket in the backlog for discussion and prioritising.